src/test/scala/thurloe/service/ThurloeServiceSpec.scala

```diff
@@ -202,16 +204,21 @@ class SwaggerServiceSpec extends FlatSpec with SwaggerService with ScalatestRout
         }
       }
 
-    Options("/swagger/thurloe.yaml") ~>
-      swaggerUiResourceRoute ~>
-      check {
-        assertResult(StatusCodes.OK) {
-          status
-        }
-        assertResult("OK") {
-          responseAs[String]
+    val pathExamples = Table("path", "/", "/swagger", "/swagger/thurloe.yaml", "/swagger/index.html", "/api",
+      "/api/thurloe/", "/thurloe")
+
+    forAll(pathExamples) { path =>
```

Comment:
This bit doesn't feel like it's part of the "Thurloe swagger docs should ..." test.